x/gamm/genesis_test.go

```diff
@@ -113,7 +113,7 @@ func TestMarshalUnmarshalGenesis(t *testing.T) {
 
 	encodingConfig := osmoapp.MakeEncodingConfig()
 	appCodec := encodingConfig.Marshaler
-	am := gamm.NewAppModule(appCodec, app.GAMMKeeper, app.AccountKeeper, app.BankKeeper)
+	am := gamm.NewAppModule(appCodec, *app.GAMMKeeper, app.AccountKeeper, app.BankKeeper)
```

Comment:
Why only deref GammKeeper? Is this just some AppModule pattern? the main keeper is derefed, but account/bank aren't? (I see the pattern repeated with other NewAppModule calls)

Reply:
Yeah, NewAppModule depends on a concrete struct for the keeper that module defines, but interfaces for everything else. We can do pointers for interfaces